src/connect/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -43,16 +40,15 @@ export function ConnectDropdownMenu<T extends string = string>(
     );
 
     return (
-        <DropdownMenu<T>
+        <DropdownMenu
+            {...dropDownProps}
+            type="controlled"
             items={dropdownItems}
             onItemClick={onItemClick}
             menuClassName={twMerge(
                 'py-3 rounded-2xl modal-shadow text-sm font-medium text-[#6F767E]',
                 menuClassName,
             )}
-            {...dropDownProps}
-        >
-            {children}
-        </DropdownMenu>
+        ></DropdownMenu>
```

Comment:
we can change this to a self-closed tag if we are not supporting `children` prop anymore